Library/Koala/Koala.swift

```diff
@@ -1790,6 +1809,9 @@ private func properties(shareContext: ShareContext,
   case let .project(project):
     result = result.withAllValuesFrom(properties(project: project, loggedInUser: loggedInUser))
     result["context"] = "project"
+  case let .liveStream(project, _):
+    result = result.withAllValuesFrom(properties(project: project, loggedInUser: loggedInUser))
+    result["context"] = "liveStream"
```

Comment:
@justinswart we typically use snake casing for property values, so should prob be `live_stream`